addons/mail/static/tests/discuss_app/discuss_tests.js

```diff
@@ -1832,21 +1838,6 @@ QUnit.test("Message shows up even if channel data is incomplete", async (assert)
     assert.containsOnce($, ".o-mail-Message:contains(hello world)");
 });
 
-QUnit.test("Create a direct message channel when clicking on start a meeting", async (assert) => {
```

Comment:
note for review: the test was duplicated at some point and already exists in `call` feature